### ISSUE TITLE
Scope UI tasks to display new data from property manager survey

### ISSUE THREAD

**Issue by anders-schneider** [CityOfDetroit/bloom#397]
For much of the data that we are collecting in the property manager survey, we already have plans for how to display it. (e.g. we're collecting information about unit types at specific AMI percentages - this will be displayed in the unit summary table UI.)

For other data (e.g. different forms of accessibility), we may need to scope work to figure out how to display this, both in the public and partners portals.

Done criteria: for every field in the property manager survey spreadsheet, we either already have a way to display that data, or we have a ticket to track the UI work to display it.

**Comment by anders-schneider** [comment]
https://github.com/CityOfDetroit/bloom/issues/396#issuecomment-909624118 has the fields in the survey

**Comment by willrlin** [comment]
So far I haven't seen much UI work that needs to be done once the survey fields are mapped to the data, except for perhaps these questions:
* Does your building offer any of the following accessibility features?
  * This question will need to be combined with the individual questions and we should decide whether we want to show it in a more structured way.
  * If we end up deciding that we will add more structure, we could then potentially map all these questions to a better structure in the backend.
* Combining the "What utilities is the resident responsible for" and "what are your average utility costs".
  * Seems like more of a backend task, once these fields are combined and put into `Listing.costsNotIncluded`, then we will still display it the same.
* What is the cost of the security deposit needed to move in?
  * Since this field is an open text field, we will need to ensure that when it's not a number, it doesn't get displayed with a dollar sign.

To summarize:
- [ ] Depending on the outcome of #67, create tasks for displaying new a11y structure.
- [ ] Either ensure that `Listing.depositMin` and `Listing.depositMax` can be displayed without `$` when not a number or add `Listing.depositDescription` field.
